packages/atlas/src/components/_navigation/SidenavBase/SidenavBase.styles.ts

```diff
@@ -13,17 +14,22 @@ export const EXPANDED_SIDENAVBAR_WIDTH = 360
 export const EXPANDED_SIDENAVBAR_WIDTH_XXS = 320
 export const NAVBAR_LEFT_PADDING = 24
 
+const setSizeNavbarWidthExpanded = css`
+  --size-sidenav-width-expanded: 320px;
+  ${media.md} {
+    --size-sidenav-width-expanded: 360px;
+  }
+`
 export const SidebarNav = styled.nav<ExpandableElementProps>`
   position: fixed;
   top: 0;
   bottom: 0;
   height: 100%;
   z-index: ${zIndex.sideNav};
-  width: ${({ expanded }) => (expanded ? `${EXPANDED_SIDENAVBAR_WIDTH_XXS}px` : 'var(--size-sidenav-width-collapsed)')};
-  ${media.md} {
-    width: ${({ expanded }) => (expanded ? `${EXPANDED_SIDENAVBAR_WIDTH}px` : 'var(--size-sidenav-width-collapsed)')};
-  }
 
+  ${setSizeNavbarWidthExpanded}
```

Comment:
It's enough to declare the var once in `SidebarNav`, all the other components should inherit the values as they descend from `SidebarNav`